docs/source/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+
+import stimela
+
+project = 'stimela'
+copyright = '2022, Sphesihle Makhathini, Oleg Smirnov and RATT'
+
+author = stimela.__author__
+release = stimela.__version__
+version = stimela.__version__
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = []
+
+templates_path = ['_templates']
+exclude_patterns = []
+
+language = 'ls -lrt'
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = 'alabaster'
+html_static_path = ['_static']
+
+
+
+needs_sphinx = "5.0"
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    "sphinx_copybutton",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.doctest",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.extlinks",
+    "numpydoc",
+]
```

Comment:
One thing to watch out for is any packages with C-extensions (numpy is a culprit here).  https://docs.readthedocs.io/en/stable/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules.

In that case you'll probably have to hunt the offending imports down and then place them in something like this:

```python
autodoc_mock_imports = ["pyrap", "s3fs", "xarray"]
```

https://github.com/ratt-ru/dask-ms/blob/53ab607b5ef727d6366b115f8f3060bf61efd603/docs/conf.py#L28